typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumVictionETHWarpConfig.ts

```diff
@@ -15,7 +16,7 @@ export const getEthereumVictionETHWarpConfig = async (
 ): Promise<ChainMap<HypTokenRouterConfig>> => {
   const viction: HypTokenRouterConfig = {
     ...routerConfig.viction,
-    ...abacusWorksEnvOwnerConfig.viction,
+    ...getOwnerConfigForAddress(abacusWorksEnvOwnerConfig.viction.owner),
```

Comment:
Why this does include `getOwnerConfigForAddress` when we removed it in the other file?

Reply:
Reverted based on discord